app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -573,7 +581,7 @@ public function handleRefundTransactionUpdate(
                         $creditmemo->setPaymentRefundDisallowed(false);
                         $creditmemo->register()->save();
                     }
-                } else if (!$this->isPartialRefundFixingMismatch($order, $transactionAmount)) { // partial refund
+                } else if (!$this->isPartialRefundFixingMismatch($order, $transactionAmount, $transaction)) { // partial refund
```

Comment:
boltTransaction ?

Reply:
@daisy1754 
Are you requesting that the name be changed?  In this method's signature, this variable is defined as `$transaction`, so there is no error here.

In the other method, the author was using the same variable name, $transaction, for two distinct objects, the Bolt transaction and the Magento transaction object.  I have never been a fan of that sort of reuse, so I changed the name to $boltTransaction for the Bolt transactions. 

The name change could be made, if that is what you are requesting.

Reply:
These objects are not the same type and were using the same variable name `$transaction` within the same method:
https://github.com/BoltApp/bolt-magento1/blob/develop/app/code/community/Bolt/Boltpay/Model/Payment.php#L473-L474

https://github.com/BoltApp/bolt-magento1/blob/develop/app/code/community/Bolt/Boltpay/Model/Payment.php#L502

The `$transaction` in 502, is actually a Bolt transaction object, 
https://github.com/BoltApp/bolt-magento1/blob/develop/app/code/community/Bolt/Boltpay/Model/Payment.php#L664-L667

where the 473-474 are Magento transaction objects.

Reply:
ah ok I misunderstood this